src/host/nodeconnections.h

```diff
@@ -213,38 +215,52 @@ namespace asynchost
           auto data_to_send = data;
           auto size_to_send = size;
 
-          // If the message is a raft append entries message, affix the
+          // If the message is a consensus append entries message, affix the
```

Comment:
My previous comment was wrong. It feels like we could unify the two paths here if we make `raft::raft_append_entries` and `pbft::pbft_append_entries` the same thing?

Reply:
As discussed this is not so simple we would need to create a common `ConsensusMsgType` and `ConsensusHeader` to put in the `append_entries` and `append_entries_response` which would mean we would need to change `recv_message` in `raft.h`. I have implemented that [here](https://github.com/microsoft/CCF/compare/playback_via_ae...consensus_ae?expand=1) to see what it looks like if anyone wants to take a look but I think it is better to keep them separate for now.

Reply:
I believe this is now resolved and implemented.